wifiphisher/common/phishinghttp.py

```diff
@@ -28,8 +28,8 @@ def get(self, url):
             self.render("index.html", **template.get_context())
             wifi_webserver_tmp = "/tmp/wifiphisher-webserver.tmp"
             with open(wifi_webserver_tmp, "a+") as log_file:
-                log_file.write('[' + T + '*' + W + '] ' + O + "GET " + T +
-                               self.request.remote_ip + W + "\n"
+                log_file.write('[' + T + '*' + W + '] ' + O + "GET" + W + " request from " + T +
```

Comment:
We use 99 characters as our maximum length so can you fix this line. All you have to do is move `self.request.path` to the next line.